docs/source/commcare.rst

```diff
@@ -140,6 +140,32 @@ It is now possible to use those values in the task actions, using syntax for cus
 - **{{trigger./data/mother/children/child_22345}}**
 - **{{trigger./data/mother/children/child_22346}}**
 
+Special Case - Accessing Atributtes
+"""""""""""""""""""""""""""""""""""
+
+It is also possible to acess values of the form atttributes.
```

Comment:
Please mention that attributes are a special case, because the trigger fields are not generated on the Tasks UI, but you can use the custom syntax to retrieve the values.